csv-data-management/extract_data_from_csv.py

```diff
@@ -395,5 +412,30 @@ def handle_params(params,file_param):
                 raise ConnectorError('Invalid File IRI {0}'.format(value))
```

Comment:
this will be caught again in line 413 and raised as /api/3/attachments not found. Do you want to handle the two separately?

Reply:
handle_params , I picked this function from VT connector, I will check if i can optimize  it

Reply:
Will work on optimization in next version